data/starfinder/paizo/core/scr_equip.lst

```diff
@@ -534,7 +534,7 @@ Personal Upgrade Mk 3	QUALITY:ItemLevel|14	COST:75000	ABILITY:Internal|AUTOMATIC
 
 
 
-Solarian Weapon	PART:1|MODIFY:Damage|SET|1d5		DAMAGE:1d5	TYPE:RESIZABLE.Weapon.Melee.Special.Solarian	PROFICIENCY:WEAPON|Solarian Weapon	BONUS:WEAPON|DAMAGESIZE|SolarianDamageProgression	SPROP:Deals Bludgeoning Damage|PREABILITY:1,CATEGORY=Internal,Solarian Weapon Damage Type ~ B	SPROP:Deals Slashing Damage|PREABILITY:1,CATEGORY=Internal,Solarian Weapon Damage Type ~ S	SPROP:Deals Piercing Damage|PREABILITY:1,CATEGORY=Internal,Solarian Weapon Damage Type ~ P	SPROP:Glows blue|PREABILITY:1,CATEGORY=Internal,Solar Manifestation Color ~ Blue		SPROP:Glows red|PREABILITY:1,CATEGORY=Internal,Solar Manifestation Color ~ Red			SPROP:Glows white|PREABILITY:1,CATEGORY=Internal,Solar Manifestation Color ~ White		SPROP:Glows yellow|PREABILITY:1,CATEGORY=Internal,Solar Manifestation Color ~ Yellow		SPROP:Solid darkness|PREABILITY:1,CATEGORY=Internal,Solar Manifestation Color ~ Black	
+Solarian Weapon	PART:1|MODIFY:Damage|SET|1d6		DAMAGE:1d6	TYPE:RESIZABLE.Weapon.Melee.Special.Solarian	PROFICIENCY:WEAPON|Solarian Weapon	BONUS:WEAPON|DAMAGESIZE|SolarianDamageProgression	SPROP:Deals Bludgeoning Damage|PREABILITY:1,CATEGORY=Internal,Solarian Weapon Damage Type ~ B	SPROP:Deals Slashing Damage|PREABILITY:1,CATEGORY=Internal,Solarian Weapon Damage Type ~ S	SPROP:Deals Piercing Damage|PREABILITY:1,CATEGORY=Internal,Solarian Weapon Damage Type ~ P	SPROP:Glows blue|PREABILITY:1,CATEGORY=Internal,Solar Manifestation Color ~ Blue		SPROP:Glows red|PREABILITY:1,CATEGORY=Internal,Solar Manifestation Color ~ Red			SPROP:Glows white|PREABILITY:1,CATEGORY=Internal,Solar Manifestation Color ~ White		SPROP:Glows yellow|PREABILITY:1,CATEGORY=Internal,Solar Manifestation Color ~ Yellow		SPROP:Solid darkness|PREABILITY:1,CATEGORY=Internal,Solar Manifestation Color ~ Black	
```

Comment:
Was this tested against a 20th level character and confirmed to grant the proper damage for both Size categories of Small and Medium? I have a feeling the damage is wrong, since I was the one who coded up this special progression track.

Reply:
Good catch.  It stays 1d6.  Though when I tried it with the 1d5 for a human at 20th it was only 5d6.  Do you get the same thing?

Reply:
I can't really test, been swamped with school work - writing a mentorship program from scratch is a fairly time consuming assignment, plus all the papers to write.  I do know that 5d6 sounds a bit low...

Reply:
You mean 6.07.08?  Anyways, don't sweat it. I'll take a quick peek under the hood and see what went wrong.

Reply:
You're not going to believe what the issue was... code was spot on, except for being "resizable". Removing that fixes the Small Sized race. However, the damage was capped by the size. Medium Weapon cannot exceed the top size. Fix was to add additional sizes above the max normal.